tests/test_connector_slack.py

```diff
@@ -78,6 +78,17 @@ class TestConnectorSlackAsync(asynctest.TestCase):
         await connector.connect()
         self.assertTrue(connector.reconnect.called)
 
+    async def test_abort_on_connection_error(self):
+        import aiohttp
+        connector = ConnectorSlack({"api-token": "abc123"})
+        connector.slacker.rtm.start = amock.CoroutineMock()
+        connector.slacker.rtm.start.side_effect = Exception()
+        connector.slacker.close = amock.CoroutineMock()
+
+        with self.assertRaises(Exception):
+            await connector.connect()
+        self.assertTrue(connector.slacker.close)
```

Comment:
Shouldn't this be `self.assertTrue(connector.slacker.close.called)`

Reply:
Whoops.  This is me being sloppy, sorry.